lib/registries/addon/drafts/draft/metadata/controller.ts

```diff
@@ -16,9 +17,10 @@ export default class RegistriesDraftMetadata extends Controller {
     model!: DraftRouteModel;
     categoryOptions = Object.keys(NodeCategory);
 
+    @alias('model.draftRegistrationManager') draftManager!: DraftRegistrationManager;
     @alias('model.draftRegistrationManager.draftRegistration') draftRegistration?: DraftRegistration;
     @alias('model.draftRegistrationManager.node') node?: NodeModel;
```

Comment:
We technically don't need these *now*. IMHO, when we *do* need them (for licenses, subjects, institutions) we should pluck them off of the `draftManager` instead of aliasing them away.

Reply:
Can confirm, licenses will only need to get the draftManager. I won't need specific aliases.